x-pack/plugins/security_solution/public/cases/components/use_create_case_modal/index.tsx

```diff
@@ -29,7 +29,7 @@ export const useCreateCaseModal = ({
   const closeModal = useCallback(() => setIsModalOpen(false), []);
   const openModal = useCallback(() => setIsModalOpen(true), []);
   const onSuccess = useCallback(
-    (theCase) => {
+    async (theCase) => {
```

Comment:
Do we need the `async` here? Do we need an `await` on one of the calls?

Reply:
`form_context.tsx` expects `onSuccess: (theCase: Case) => Promise<void>` and waits for the `onSuccess` to be fulfilled. Do you think it shouldn't?